qiskit/algorithms/__init__.py

```diff
@@ -72,71 +72,57 @@
    MaximumLikelihoodAmplitudeEstimationResult
 
 
-Eigen Solvers
--------------
-
-Algorithms to find eigenvalues of an operator. For chemistry these can be used to find excited
-states of a molecule, and qiskit-nature has some algorithms that leverage chemistry specific
-knowledge to do this in that application domain. These algorithms are pending deprecation.
-One should instead make use of the Eigensolver classes in the section below, which leverage
-Runtime primitives.
-
-.. autosummary::
-   :toctree: ../stubs/
-   :nosignatures:
-
-   Eigensolver
-   EigensolverResult
-
-.. autosummary::
-   :toctree: ../stubs/
-   :nosignatures:
-
-   NumPyEigensolver
-   VQD
-   VQDResult
-
 Eigensolvers
 ------------
 
 Algorithms to find eigenvalues of an operator. For chemistry these can be used to find excited
-states of a molecule, and qiskit-nature has some algorithms that leverage chemistry specific
+states of a molecule, and ``qiskit-nature`` has some algorithms that leverage chemistry specific
 knowledge to do this in that application domain.
 
+Primitive-based Eigensolvers
+++++++++++++++++++++++++++++
+
+These algorithms are based on the Qiskit Primitives, such as :class:`.Estimator`.
+This execution paradigm replaces the use of :class:`.QuantumInstance` in the algorithms
+and in future, the primitive-based Eigensolvers should be used.
+
 .. autosummary::
    :toctree: ../stubs/
 
    eigensolvers
 
-Evolvers
---------
 
-Pending deprecation: This package has been superseded by the package below. It will be
-deprecated in a future release and subsequently removed after that:
+Legacy Eigensolvers (pending deprecation)
```

Comment:
I would avoid saying pending deprecation, that doesn't mean anything to users. Deprecation has a meaning, we've marked this feature for removal in a future release and it will only continue to work as is until that removal but pending deprecation isn't actionable. Instead we should just say the legacy interface has been largely superseded by the primitives version but is still supported for now

Reply:
Yeah that makes sense -- changed in 9b45d78 ! 🙂